### DISCUSSION TITLE
SERIOUS SECURITY WARNING ON WINDOWS

### DISCUSSION THREAD

**Discussion by facebookCtuTien** [community/community#122750] [Other Feature Feedback, Questions, & Ideas]
Tonight, when I checked my Microsoft account, I discovered that I CANNOT log out of my Microsoft account from the Edge browser, as my account still has Administrator rights,  and deleting the administrator account is impossible!
  I got all my account passwords extremely easily, (including Chrome by entering passwords from another browser), which is completely absurd when I have enabled Chrome app lock and use more  Chrome add-on to prevent other people at the company from accessing all my passwords.

What if hackers create an .exe containing a small malicious code that can directly take away the password data file??

**Comment by goldieowner17** [comment]
Hi @facebookCtuTien ,

Thanks for being a part of the GitHub Community, we're glad you're here

The Discover category is for conversations related to your GitHub features etc. I have moved this post to General.

We want to make sure you're getting the best support you can and that your message goes where you intended, but this may not be the right place.


Best of luck!

**Comment by ebndev** [comment]
Hi @facebookCtuTien!

As @goldieowner17 mentioned, we want you to get the best support that you can and while it's possible that another GitHub user might have run into this same issue, the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas.

You might want to try asking for help somewhere that focuses on questions about Microsoft Edge or support for your Microsoft account.
